raiden/tests/integration/test_pythonapi.py

```diff
@@ -402,10 +402,10 @@ def test_payment_timing_out_if_partner_does_not_respond(  # pylint: disable=unus
     app0, app1 = raiden_network
     token_address = token_addresses[0]
 
-    def fake_receive(room, event):  # pylint: disable=unused-argument
-        return True
+    def fake_send(room, event):  # pylint: disable=unused-argument
+        return
 
-    with patch.object(app1.raiden.transport, "_handle_message", side_effect=fake_receive):
+    with patch.object(app1.raiden.transport, "_send_raw", side_effect=fake_send):
```

Comment:
umm... what happened here? how did this change all of a sudden from handling receiving a message to `_send_raw`?

Reply:
The test patching didn't work as expected, when a room was created during healthcheck, the payment did actually succeed (at least the greenlet.value() wasnt gone after timeout). I changed it _send_raw, as the test seems to be supposed to test what happens if the partner doesn't respond. Not sending a message seems like the appropriate thing to do for "not respond".